README.md

```diff
@@ -84,6 +84,7 @@ Supported datasets:
 - [x] [AI Challenger](https://github.com/AIChallenger/AI_Challenger_2017)
 - [x] [OCHuman](https://github.com/liruilong940607/OCHumanApi)
 - [x] [CrowdPose](https://github.com/Jeff-sjtu/CrowdPose)
+- [x] [sub-JHMDB](http://jhmdb.is.tue.mpg.de/dataset)
```

Comment:
What does `sub-` mean?

Reply:
Only a subset of JHMDB

Reply:
JHMDB is initially proposed for action recognition. Some works also use a subset of JHMDB (sub-JHMDB) for pose estimation.

Reply:
Good to know. Put it in the intro of preparation will be helpful